core/kazoo_auth/Makefile

```diff
@@ -0,0 +1,49 @@
+ROOT = ../..
+PROJECT = kazoo_auth
+
+all: compile
+
+# Target to be built by `compile` and `compile-test`
+COMPILE_MOAR = priv/kz_auth_rsa_drv.so
+CLEAN_MOAR = clean-so clean-c_src-env
+
+include $(ROOT)/make/kz.mk
+
+ifeq ($(PLATFORM),darwin)
+	CC ?= cc
+	CFLAGS ?= -O3 -std=c99 -arch x86_64 -Wall -Wmissing-prototypes -fPIC -shared
+	LDFLAGS ?= -arch x86_64 -flat_namespace -undefined suppress
+else ifeq ($(PLATFORM),freebsd)
+	CC ?= cc
+	CFLAGS ?= -O3 -std=c99 -finline-functions -Wall -Wmissing-prototypes -fPIC -shared
+else ifeq ($(PLATFORM),linux)
+	CC ?= gcc
```

Comment:
Why different linux/bsd targets? Is forcing gcc instead of clang important? (cc resolves to clang on my machines)